commands/channel.go

```diff
@@ -345,32 +350,35 @@ func makeChannelPrivateCmdF(c client.Client, cmd *cobra.Command, args []string)
 }
 
 func renameChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	var newDisplayName, newChannelName string
+	existingTeamChannel := args[0]
+	newChannelName, _ := cmd.Flags().GetString("name")
```

Comment:
I see that rename team also ignores the errors. Do you know why ?

Reply:
I am not able to find where but i think @mgdelacroix suggested that if there was an error in getting flag, then the flag variable would be empty and error will be thrown in the next code step. Anyways what do you suggest

Reply:
I think that might clash with explicitly passing an empty value. In this case, we need a value, so this is fine. But in some cases, we might have a use case for passing empty value. In that case, that check is necessary. And I do see the error being checked in some parts of the codebase, so for the sake of consistency I suggest to add the check.

Reply:
got it

Reply:
i will add now

Reply:
But i also see in many places it is being checked to see if flag is being empty. What error message should it be here in our case ?